### PR TITLE
[C-API] fix invalid parameter direction

### PR DESCRIPTION
**Pull request by gichan-jang**
Fix invalid parameter direction of the C-API.

Signed-off-by: gichan-jang <gichan2.jang@samsung.com>

Self evaluation:
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by again4you — APPROVED** [review]
LGTM! We don't need to do ACR for this patch.

**Review by dongju-chae — APPROVED** [review]
LGTM

File: api/capi/include/nnstreamer.h [hunk 1]
```diff
@@ -223,17 +223,17 @@ typedef enum {
  * @since_tizen 5.5
  * @remarks The @a data can be used only in the callback. To use outside, make a copy.
  * @remarks The @a info can be used only in the callback. To use outside, make a copy.
- * @param[out] data The handle of the tensor output (a single frame. tensor/tensors). Number of tensors is determined by ml_tensors_info_get_count() with the handle 'info'. Note that the maximum number of tensors is 16 (#ML_TENSOR_SIZE_LIMIT).
- * @param[out] info The handle of tensors information (cardinality, dimension, and type of given tensor/tensors).
- * @param[out] user_data User application's private data.
+ * @param[in] data The handle of the tensor input (a single frame. tensor/tensors). Number of tensors is determined by ml_tensors_info_get_count() with the handle 'info'. Note that the maximum number of tensors is 16 (#ML_TENSOR_SIZE_LIMIT).
```
Comment (myungjoo) [inline api/capi/include/nnstreamer.h:226]: Sorry. I was a bit confused in a previous comment. This is ```... [in] ... tensor output of the pipeline```.

Comment (gichan-jang) [reply api/capi/include/nnstreamer.h]: Thanks. Updated.

**Review by myungjoo — APPROVED** [review]
LGTM

**Pull request merged**